lib/qesapdeployment.pm

```diff
@@ -1273,7 +1273,8 @@ sub qesap_cluster_log_cmds {
 sub qesap_cluster_logs {
     my $provider = get_required_var('PUBLIC_CLOUD_PROVIDER');
     my $inventory = qesap_get_inventory(provider => $provider);
-    if (script_run("test -e $inventory") == 0)
+    my $ret = script_run("test -e $inventory", 90);
+    if ($ret == 0)
```

Comment:
No need for the extra `$ret` variable nor to split it in 2 lines if only increasing the timeout.

Reply:
fixed it